requirements.txt

```diff
@@ -10,3 +10,6 @@ geopack>=1.0.10
 scipy
 hapiclient>=0.2.2
 viresclient
+fsspec
```

Comment:
`ffspec`, `s3fs` (and `aioboto3` though hopefully we can get cdflib to deal with it) should go into the `setup.py` file since requirements.txt doesn't ever get called during setup.